kalite/distributed/static/js/distributed/exercises.js

```diff
@@ -1417,7 +1417,7 @@ window.ExerciseQuizView = Backbone.View.extend({
                     item: "/api/store/storeitem/gift_card/",
                     purchased_at: window.statusModel.get_server_time(),
                     reversible: false,
-                    context_id: ds.ab_testing.unit || 0,
```

Comment:
This is a rather rubbish way we implemented of having points for Quizzes. Might be better to just put points on QuizLogs instead?